### PR TITLE
added changes to flask-server to configure admin panel

### PR DESCRIPTION
**Pull request by carpecodeum**
Please provide enough information so that others can review your pull request:

added changes to flask-server to configure the admin panel on the client side

Explain the **details** for making this change. What existing problem does the pull request solve?
changes in the helpers.py and stories.controller.py allow the endpoint api/stories/all to be served only when the user is admin
also updated the read me to allow .env file to incorporate the credentials for admin.
**Test plan (required)**
tested using postman that the data of all the stories is served only when the user is an admin

**Closing issues** #470

### CODE REVIEW

**Comment by carpecodeum** [comment]
@Anmolbansal1 the build is failing because the generate_password_hash method takes a not null value and i have imported an env variable for admin_password and thus stays null during the build. Please guide me as to what i should do now?

**Comment by carpecodeum** [comment]
@Anmolbansal1 the build is failing because the generate_password_hash method takes a not null value and i have imported an env variable for admin_password and thus stays null during the build.
do you have any suggestions on what should i do?

**Comment by carpecodeum** [comment]
@Anmolbansal1 what are your views on this?

**Comment by carpecodeum** [comment]
@Anmolbansal1 thinking of making a script for creating admins. What say?

File: fact-bounty-flask/requirements.txt [hunk 1]
```diff
@@ -43,7 +43,7 @@ SQLAlchemy==1.2.18
 tornado==6.0
 tweepy==3.8.0
 typed-ast==1.2.0
-urllib3==1.24.1
+urllib3==1.23.0
```
Comment (Anmolbansal1) [inline fact-bounty-flask/requirements.txt]: Why is this package version changed??

File: fact-bounty-flask/migrations/versions/02db7a3aaa95_.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
+"""empty message
```
Comment (Anmolbansal1) [inline fact-bounty-flask/migrations/versions/02db7a3aaa95_.py]: I think that two migrations files are unnecessary as only a new column is added to user table, so try to generate only corresponding to that change

File: fact-bounty-flask/api/commands.py [hunk 3]
```diff
@@ -168,3 +191,34 @@ def deploy():
...
+            name=admin_username,
+            email=admin_email,
+            password=admin_password,
+            role=admin_role,
+        )
+        user.save()
+    except Exception as e:
+        print(str(e))
+        response = {"message": "Something went wrong!!"}
+        return make_response(jsonify(response)), 500
```
Comment (Anmolbansal1) [inline fact-bounty-flask/api/commands.py]: It's unclear why this method return a response like a route, it's just a click command. Also please attach some screenshot demonstrating this change.

File: fact-bounty-flask/api/commands.py [hunk 4]
```diff
@@ -6,16 +6,39 @@
...
+regex to check valid email is entered
+"""
+regex = r"^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$"
+
+"""
+function for
+for validating an Email
+"""
+
+
```
Comment (Anmolbansal1) [inline fact-bounty-flask/api/commands.py]: Please remove these unnecessary black lines

File: fact-bounty-flask/api/stories/controller.py [hunk 5]
```diff
@@ -4,41 +4,49 @@
...
     @swag_from("../../docs/stories/get_all.yml")
     def get(self):
+        user_id = get_jwt_identity()
+        user = model.User.find_by_user_id(user_id)
 
-        es_index = current_app.config["ES_INDEX"]
-        es = current_app.elasticsearch
+        if user.role == "admin":
+            es_index = current_app.config["ES_INDEX"]
+            es = current_app.elasticsearch
```
Comment (Anmolbansal1) [inline fact-bounty-flask/api/stories/controller.py]: This file changes looks good, could you attach screenshots on request this url with admin user, you may use postman :smile:

File: fact-bounty-flask/api/app.py [hunk 6]
```diff
@@ -15,28 +15,23 @@
...
-        def create_tables():
-            db.create_all()
+    @app.before_first_request
+    def create_tables():
+        db.create_all()
 
-        return app
-    except Exception as err:
-        print("Error occured:", err)
+    return app
```
Comment (Anmolbansal1) [inline fact-bounty-flask/api/app.py]: Why have you removed this error handling of flask app??

File: fact-bounty-flask/api/stories/controller.py (hunk 5 above)
Comment (carpecodeum) [reply fact-bounty-flask/api/stories/controller.py]: sure!

File: fact-bounty-flask/requirements.txt (hunk 1 above)
Comment (carpecodeum) [reply fact-bounty-flask/requirements.txt]: this was changed as a result of a warning coming during the setup.

**Pull request closed without merging**